src/components/ApplicationMenu/ApplicationMenu.js

```diff
@@ -150,12 +158,24 @@ class ApplicationMenu extends Component<Props> {
     // During onboarding, there is no selected project (because none exists
     // yet). Therefore, we only want to show the 'Project' menu when a project
     // is selected.
-    if (selectedProjectId) {
+    if (selectedProject) {
       // The `Project` menu should be inserted right after `Edit`, which will
       // have a different index depending on the platform.
       const editMenuIndex = template.findIndex(menu => menu.id === 'edit');
 
-      let submenu = [];
+      let submenu = [
+        {
+          label: isMac ? 'Open in Finder' : 'Open in Explorer',
```

Comment:
As mentioned in #52 it would be also OK to have the label `Open Folder`.

If you'd like to have the name 'Finder' / 'Explorer' then we should have a fallback to `Open Folder`. 
Because for Linux `Open in Explorer` is strange as it's called `Nautilus` in Ubuntu - and many more names as it is different at each distribution. For OpenSUSE I think it's called Dolphin.

Should we move that caption to `platform.service`? As it's also needed in the UI just if we're using Finder/Explorer. Open Folder could be hard coded at both locations.

Reply:
Yeah, good call. I was just being lazy, but you're right, we should make it good for Linux as well